src/display/annotation_layer.js

```diff
@@ -501,7 +501,7 @@ class AnnotationElement {
    *
    * @public
    * @memberof AnnotationElement
-   * @returns {HTMLSectionElement|Array<HTMLSectionElement>}
+   * @returns {HTMLElement|Array<HTMLElement>} A section element or an array of section elements
```

Comment:
I cannot imagine that this passes linting, since this line is now too long; did you forget to run `gulp lint` locally?
```javascript
 * @returns {HTMLElement|Array<HTMLElement>} A section element or
 *   an array of section elements.
```

Reply:
Honestly, I did not lint, this is only done through GitHub UI.

Definitely would have done so, if it wasn't just small JSDoc changes. Did not think `lint` would have a problem with comment changes.

Do you have a suggestion on how this could be shortened? What do you think about `Section element(s)` ?

Reply:
> Honestly, I did not lint, this is only done through GitHub UI.

Please note that we don't recommend using that, please find the recommended work-flow outlined in https://github.com/mozilla/pdf.js/wiki/Contributing

In particular, https://github.com/mozilla/pdf.js/wiki/Squashing-Commits probably won't work if using the GitHub UI.

> Do you have a suggestion on how this could be shortened?

Well, https://github.com/mozilla/pdf.js/pull/15143#discussion_r915870544 already shows how to fix that :-)

Reply:
> Please note that we don't recommend using that, please find the recommended work-flow outlined in https://github.com/mozilla/pdf.js/wiki/Contributing

Did a quick read-through before but thanks for the reminder! 👍 Unfortunately, if I followed as a whole, I wouldn't have made the PR, as I do not have the capacity for all that rn. 

> In particular, https://github.com/mozilla/pdf.js/wiki/Squashing-Commits probably won't work if using the GitHub UI.

I'm pretty new to contribution as a whole, but [GH should be able](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests) *(I'm guessing it's allowed)* to squash a PR, if not, I should be able to check the generated stuff out and do it manually, but I'll definitely keep it in mind for the future.

> Well, https://github.com/mozilla/pdf.js/pull/15143#discussion_r915870544 already shows how to fix that :-)

Yeah... realized a minute later.. hahha

Reply:
Squashed them :)